scripts/techsupport

```diff
@@ -47,10 +47,14 @@ if [[ $1 == 'reset' ]]; then
 else
   top -n 10 -b > top-output.txt
   rsync -a /home/pi/.firewalla ./ --exclude run/
+  rsync -a /home/pi/.router ./ --exclude run/ --exclude tmp/ --exclude config/sshd/keys
```

Comment:
From rsync help
> -a, --archive               archive mode; equals -rlptgoD (no -H,-A,-X)

With "-a" option, if run as non-root user, there will be error like this
```
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1207) [sender=3.1.3]
```

Safe to use `-rlpt` instead.